os.gemspec

```diff
@@ -46,18 +46,15 @@ Gem::Specification.new do |s|
     s.specification_version = 4
 
     if Gem::Version.new(Gem::VERSION) >= Gem::Version.new('1.2.0') then
-      s.add_runtime_dependency(%q<os>.freeze, [">= 0"])
```

Comment:
The os gem can't depend on itself.  I think this is part of the messy legacy of `jeweler`.

Reply:
Yeah I think jeweler was auto inserting this, scary...